From e16b5ff97eb7be7c5f37f7fbbf43aa4db36a3a80 Mon Sep 17 00:00:00 2001
From: Florian Walch <florian.walch@franka.de>
Date: Wed, 27 Sep 2017 17:33:12 +0200
Subject: [PATCH 1/2] Added install rule for package.xml

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 02022a7..f3dc28d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -117,6 +117,9 @@ install(EXPORT FrankaTargets
   DESTINATION ${INSTALL_CMAKE_CONFIG_DIR}
 )
 
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/libfranka)
+
 ## Subprojects
 
 # Ignore find_package(Franka) in subprojects.
-- 
2.7.4

