From 1b7e89202c17a5dd376945fb544ff1d0eb95fec7 Mon Sep 17 00:00:00 2001
From: Florian Walch <florian.walch@franka.de>
Date: Wed, 27 Sep 2017 17:33:12 +0200
Subject: [PATCH 1/2] Added install rule for package.xml

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9c97e95..39d0c26 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -153,6 +153,9 @@ install(EXPORT FrankaTargets
   DESTINATION ${INSTALL_CMAKE_CONFIG_DIR}
 )
 
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/libfranka)
+
 ## Subprojects
 
 # Ignore find_package(Franka) in subprojects.
-- 
2.25.1

