From 5c915e749d0bca7b494eb33d07b6be28395e1461 Mon Sep 17 00:00:00 2001
From: Florian Walch <florian.walch@franka.de>
Date: Wed, 27 Sep 2017 17:34:44 +0200
Subject: [PATCH 1/4] Customized RPM template file

---
 rpm/template.spec.em | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/rpm/template.spec.em b/rpm/template.spec.em
index 259dcdd..ee56bc7 100644
--- a/rpm/template.spec.em
+++ b/rpm/template.spec.em
@@ -44,6 +44,9 @@ mkdir -p obj-%{_target_platform} && cd obj-%{_target_platform}
     -DCMAKE_INSTALL_LIBDIR="lib" \
     -DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
     -DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
+    -DBUILD_DOCUMENTATION=ON \
+    -DBUILD_EXAMPLES=OFF \
+    -DBUILD_TESTS=OFF \
     -DSETUPTOOLS_DEB_LAYOUT=OFF \
     -DCATKIN_BUILD_BINARY_PACKAGE="1" \
     ..
-- 
2.25.1

