From 5bdf657e1870b282c7d42d1e1339d0f47b204c27 Mon Sep 17 00:00:00 2001
From: Florian Walch <florian.walch@franka.de>
Date: Fri, 10 Aug 2018 16:45:28 +0200
Subject: [PATCH 2/2] Install into catkin locations

---
 CMakeLists.txt          | 11 +++++------
 examples/CMakeLists.txt |  5 ++---
 2 files changed, 7 insertions(+), 9 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0184d16..c275a2a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -97,7 +97,7 @@ endif()
 
 target_include_directories(franka PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-  $<INSTALL_INTERFACE:include>
+  $<INSTALL_INTERFACE:include/libfranka>
 )
 
 target_link_libraries(franka PRIVATE
@@ -109,15 +109,14 @@ target_link_libraries(franka PRIVATE
 )
 
 ## Installation
-include(GNUInstallDirs)
-set(INSTALL_CMAKE_CONFIG_DIR ${CMAKE_INSTALL_LIBDIR}/cmake/Franka)
+set(INSTALL_CMAKE_CONFIG_DIR share/franka/cmake)
 
 install(TARGETS franka
   EXPORT FrankaTargets
-  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION lib/libfranka
 )
-install(DIRECTORY include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+install(DIRECTORY include/ DESTINATION include/libfranka)
 
 export(EXPORT FrankaTargets
   NAMESPACE Franka::
diff --git a/examples/CMakeLists.txt b/examples/CMakeLists.txt
index 341f80c..d810d02 100644
--- a/examples/CMakeLists.txt
+++ b/examples/CMakeLists.txt
@@ -46,8 +46,7 @@ endforeach()
 target_link_libraries(joint_impedance_control Threads::Threads)
 target_link_libraries(motion_with_control Poco::Foundation)
 
-include(GNUInstallDirs)
 install(TARGETS ${EXAMPLES}
-  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION lib/libfranka
 )
-- 
2.25.1

