From 7cf7c3a771b7f352ed671aa43f7870b9015113c6 Mon Sep 17 00:00:00 2001
From: Florian Walch <florian.walch@franka.de>
Date: Mon, 2 Oct 2017 15:49:01 +0200
Subject: [PATCH 3/3] Disable documentation

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 6c3eb0a..d908fcd 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -31,7 +31,7 @@ override_dh_auto_configure:
 		-DCATKIN_BUILD_BINARY_PACKAGE="1" \
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
 		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
-		-DBUILD_DOCUMENTATION=ON \
+		-DBUILD_DOCUMENTATION=OFF \
 		-DBUILD_EXAMPLES=OFF \
 		-DBUILD_TESTS=ON
 
-- 
2.7.4

